cli.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+console.log('Generating image sprite ...')
+const runner = require('./runner')
+
+runner.run()
+  .then(console.log('Done'))
+  .catch(error => console.error(error))
```

Comment:
This can be shortened to

```suggestion
  .catch(console.error)
```